tools/pathwaymatcher/pathwaymatcher.xml

```diff
@@ -0,0 +1,381 @@
+                      <option value="ensembl">Proteins - Ensembl identifier list</option>
+                      <option value="proteoforms">Proteoforms</option>
+                  </param>
+
+                  <!-- Genetic variants -->
+                  <when value="rsid">
+                      <param format="txt" name="input_rsid" type="data" label="SNP rsId list"
+                          help="The file contains one rsid identifier as defined in dbSNP[1] on each row.
+                          The list must be ordered by chromosome and base pair (bp). The list must not have duplicates.
+                          All rsids must appear in the human assembly GRCh37.p13. "/>
+                  </when>
+
+                  <when value="chrbp">
+                      <param format="txt" name="input_chrbp" type="data" label="Chromosomes and base pairs"
+                          help="Genetic variants can also be represented using the chromosome and the base pair numbers.
+                          The input should be sorted by chromosome number and then by base pair.  "/>
+                  </when>
+
+                  <when value="vcf">
+                      <param format="txt" name="input_vcf" type="data" label="Variant Call Format Specification"
+                          help="The input follows the Variant Call Format Specification[2] v4.3.
+                          It also allows the possibility to specify only the first 4 columns in the data section of the file:
+                          CHROM, POS, ID, REF.  "/>
+                  </when>
+
+                  <!-- Genes -->
+                  <when value="gene">
+                      <param format="txt" name="input_gene" type="data" label="Genes"
+                          help="File with a one gene name in each line. Genes follow the HUGO gene nomenclature[3]."/>
+                  </when>
+
+                  <!-- Peptides  -->
+                  <when value="peptide">
+                      <param format="txt" name="input_peptide" type="data" label="Simple list"
+                          help="File with a one peptide sequence in each line. You also need to choose a FASTA file to search them against it."/>
+
+                      <param format="fasta" name="input_peptide_database" type="data" label="Protein Database"
+                          help="Select FASTA database from history"/>
+
+                      <param name="ptm_range" type="integer" value="0" label="PTM position range" optional="true"
+                          help="Plus minus positions for the same PTM site."/>
+                  </when>
+
+                  <when value="modifiedpeptide">
+                      <param format="txt" name="input_modifiedpeptide" type="data" label="Peptide List with PTM types and sites"
+                         help="Each line of the file corresponds to a single peptide with post-translational modifications. You also need to choose a FASTA file to search them against it."/>
```

Comment:
the sentence with the fasta file is redundant, as the user can not submit without it